tests/Helpers/JWKFetcherTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Auth0\Tests\Helpers;
+
+use Auth0\SDK\Helpers\JWKFetcher;
+use josegonzalez\Dotenv\Loader;
+use GuzzleHttp\Exception\RequestException;
+
+/**
+ * Class JWKFetcherTest
+ *
+ * @package Auth0\SDK\Helpers
+ */
+class JWKFetcherTest extends \PHPUnit_Framework_TestCase
+{
+
+    /**
+     * Issuer for JWKS URL.
+     *
+     * @var string
+     */
+    public static $issuer;
+
+    /**
+     * Setup before class.
+     *
+     * @return void
+     */
+    public static function setUpBeforeClass()
+    {
+        $env_path = '.env';
+        if (file_exists($env_path)) {
+            $loader = new Loader($env_path);
+            $loader->parse()->putenv(true);
+        }
+
+        self::$issuer = 'https://'.getenv('DOMAIN').'/';
```

Comment:
rename to domain / base uri?

Reply:
No need, I'm using it as an issuer in this case.

Reply:
it's confusing since you renamed the param on `fetch_keys ` from `iss` to `domain`. But /shrug